packages/twenty-server/src/engine/core-modules/user-workspace/user-workspace.service.ts

```diff
@@ -196,4 +200,51 @@ export class UserWorkspaceService extends TypeOrmQueryService<UserWorkspace> {
       ),
     }));
   }
+
+  async getUserWorkspaceForUser({
```

Comment:
```suggestion
  async getUserWorkspaceForUserOrThrow({
```
to follow your pattern below?